.vscode/launch.json

```diff
@@ -10,15 +10,8 @@
       "cwd": "${workspaceFolder}/backend",
       "jinja": true,
       "envFile": "${workspaceFolder}/.env",
-      "osx": {
-        "python": "${workspaceFolder}/backend/.venv/bin/python"
-      },
-      "linux": {
-        "python": "${workspaceFolder}/backend/.venv/bin/python"
-      },
-      "windows": {
-        "python": "${workspaceFolder}/backend/.venv/Scripts/python.exe"
-      }
+      // use a settings.json file to set this value.
```

Comment:
please add a `setting.json` file  with

```
{"pythonPath"': ".venv"}

Reply:
The old code is a standard setup for virtual environments and includes information that is platform specific.

Reply:
Problem is it's not correct, at least not for osx. For example, if I'm using poetry and poetry shell, then it's going to create a folder for my environment. Changing launch.json seems wrong. settings.json is currently gitignore'd, so I figure every dev can set up their python location using that, then launch.json doesn't need to change. thoughts?

Reply:
The instructions in the README are intended to explain how to use this launch.json.  Debugging does work on my linux machine using these instructions.   As far as I remember, the problem was that poetry installs the environment in the root folder by default.  
https://github.com/Incredihire/Survey?tab=readme-ov-file#vs-code-debugging

Reply:
Configuring poetry venv as python interpreter in VS Code's setting seem to work. Doing this, you won't have to set python path from  `launch.json` and `settings.json`. The backend debugger configuration would be just:

```json
    {
      "name": "Debug Backend: Launch FastAPI Python Debugger",
      "type": "debugpy",
      "request": "launch",
      "module": "uvicorn",
      "args": ["app.main:app", "--reload"],
      "cwd": "${workspaceFolder}/backend",
      "jinja": true,
      "envFile": "${workspaceFolder}/.env"
    },
```